app/src/main/java/org/covidwatch/android/ble/EnableBluetoothService.kt

```diff
@@ -0,0 +1,83 @@
+package org.covidwatch.android.ble
+
+import android.app.*
+import android.bluetooth.BluetoothAdapter
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.core.content.ContextCompat
+import org.covidwatch.android.MainFragment.Companion.INITIAL_VISIT
+import org.covidwatch.android.R
+import org.covidwatch.android.ui.MainActivity
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+class EnableBluetoothService : IntentService("EnableBluetoothService") {
+
+    private val CHANNEL_ID = "EnableBluetoothServiceChannel"
+    private val NOTIF_ID = 15
+    private val POLLING_INTERVAL = 5L
```

Comment:
For production I think we should use a much longer interval. If I turn bluetooth off i probably have a reason to do so and I wouldn't want to be pinged every 5 minutes to turn it back on. Maybe once every few hours or per day!?